packages/app-mobile/components/ProfileSwitcher/ProfileSwitcher.tsx

```diff
@@ -0,0 +1,40 @@
+const React = require('react');
+import { useMemo } from 'react';
+const { View, Text, StyleSheet } = require('react-native');
+import createRootStyle from '../../utils/createRootStyle';
+import ScreenHeader from '../ScreenHeader';
+import { _ } from '@joplin/lib/locale';
+const { ActionButton } = require('../action-button');
+// const ReactNativeActionButton = require('react-native-action-button').default;
+
+interface Props {
+	themeId: number;
+}
+
+const useStyle = (themeId: number) => {
+	return useMemo(() => {
+		return StyleSheet.create({
+			...createRootStyle(themeId),
+		});
+	}, [themeId]);
+};
+
+export default (props: Props) => {
+	const style = useStyle(props.themeId);
+
+	return (
+		<View style={style.root}>
+			<ScreenHeader title={_('Edit notebook')} showSaveButton={false} showSideMenuButton={false} showSearchButton={false} />
+			<View><Text>TEST</Text></View>
+			<ActionButton mainButtonPress={() => { console.info('AAAAAAAAAAAAAAAAAAAAAAA'); }} mainButton={
```

Comment:
Is this log message on purpose (and/or the final one)? Looks like a quick debug message to me.

Reply:
Yes it's just for testing

Reply:
Since tbe PR is not in draft status, it looked a bit off to me. ;-) Thanks for the info.